CRM/Upgrade/Incremental/php/FourSeven.php

```diff
@@ -287,6 +287,16 @@ public function upgrade_4_7_15($rev) {
     $this->addTask(ts('Upgrade DB to %1: SQL', array(1 => $rev)), 'runSql', $rev);
   }
 
+  /**
+   * Upgrade function.
+   *
+   * @param string $rev
+   */
+  public function upgrade_4_7_16($rev) {
+    $this->addTask(ts('Upgrade DB to %1: SQL', array(1 => $rev)), 'runSql', $rev);
+    $this->addTask('Add new CiviMail fields', 'addMailingTemplateType');
```

Comment:
@totten  Just want to point out that you could save yourself the trouble of adding that `addMailingTemplateType` callback function by using the generic `addColumn` callback. See some examples above.